libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLine.kt

```diff
@@ -1308,10 +1341,13 @@ internal class MapRouteLine(
          * Calculates line segments based on the legs in the route line and color representation
          * of the traffic congestion. The items returned can be used to create a style expression
          * which can be used to style the route line. The styled route line will be colored
-         * according to the traffic conditions indicated in the @param route.
+         * according to the traffic conditions indicated in the @param route. For any road class
+         * included in the @param trafficBackfillRoadClasses 'unknown' traffic values will be
+         * colored with the low traffic color instead of the color configured for unknown traffic
+         * congestion.
```

Comment:
```suggestion
         * according to the traffic conditions indicated in the @param route. For any road class
         * included in the @param trafficBackfillRoadClasses, all route segments with an 'unknown' traffic congestion annotation and a matching road class will be
         * colored with the low traffic color instead of the color configured for unknown traffic
         * congestion.
```